#Details of service data types
float64 changeLSpeed
---
bool response
